pkg/volume/util/subpath/subpath_windows.go

```diff
@@ -86,6 +86,17 @@ func isLinkPath(path string) (bool, error) {
 	return false, nil
 }
 
+// Escape the special character in vsphere windows path
+func escapeWindowsPath(path string) (string) {
```

Comment:
this might be a little complicated considering different system might allow different characters. trying to check information related to this

@andyzhangx do you know whether there is a generic way to process special character in Windows?

Reply:
Good suggestion! I checked the doc for the `Get-Item` powershell command and found out we can change to use `-LiteralPath` instead of using `-Path` in order to prevent powershell to interpret the special characters like `[]` as wildcards, and when using `-LiteralPath`, the string will be interpreted as it is. e.g. vsphere path `'C:\var\lib\kubelet\plugins\kubernetes.io\vsphere-volume\mounts\[datastore_name] kubevols'` can be correctly interpreted in this case.
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-item?view=powershell-7.1